public/css/style.css

```diff
@@ -292,14 +308,14 @@ input::-webkit-calendar-picker-indicator {
 .body-layout {
     min-height: 63vh;
     margin-top: 100px;
-    margin-bottom: 100px;
+    margin-bottom: 50px;
```

Comment:
Do not modify this since there are multiple pages using this class, please find a work around

Reply:
You can do something like `.calendar-container.body-layout { margin-bottom: 50px; }` since `.calendar-container.body-layout` is more sepacific than only `.body-layout`, it will overwrite the old value